### PR TITLE
Vecs Adapters Blog Post

### PR DESCRIPTION
**Pull request by olirice**
## What kind of change does this PR introduce?
LW8 Friday blog post for supbase/vecs adapters feature

### CODE REVIEW

File: apps/www/_blog/2023-08-12-vecs-adapters.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+date: '2023-08-11'
+toc_depth: 3
+---
+
+[supabase/vecs](https://github.com/supabase/vecs) is our python client for rapid development of vector projects based on postgres + pgvector.
+
+Today, we're releasing a new feature for vecs called "adapters" that enables developers to manage embeddings without ever having to touch a vector.
+Adapters allow you to associate transforms with a collection (table of vectors) that are applied to your input when upserting and querying.
+
+As an example, lets look at how we might adapt text input.
```
Comment (gregnr) [inline apps/www/_blog/2023-08-12-vecs-adapters.mdx:20]:
```suggestion
As an example, let's look at how we might adapt text input.
```

**Review by gregnr — COMMENTED** [review]
🎉

File: apps/www/_blog/2023-08-12-vecs-adapters.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  - postgres
+  - AI
+  - python
+date: '2023-08-11'
+toc_depth: 3
+---
+
+[supabase/vecs](https://github.com/supabase/vecs) is our python client for rapid development of vector projects based on postgres + pgvector.
+
+Today, we're releasing a new feature for vecs called "adapters" that enables developers to manage embeddings without ever having to touch a vector.
```
Comment (kiwicopple) [inline apps/www/_blog/2023-08-12-vecs-adapters.mdx:17]: > "without ever having to touch a vector."

I'm wondering if they ever had to touch a vector previously? Perhaps, "without ever having to touch an external lib" (needs rephrasing, but hopefully makes the benefit clearer)

File: apps/www/_blog/2023-08-12-vecs-adapters.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+Adapters allow you to associate transforms with a collection (table of vectors) that are applied to your input when upserting and querying.
```
Comment (kiwicopple) [inline apps/www/_blog/2023-08-12-vecs-adapters.mdx:18]: this sentence is a bit opaque, perhaps something like:

"Adapters transform your input into a new format when upserting and querying. For example, you can transform text into embeddings, or ...."

File: apps/www/_blog/2023-08-12-vecs-adapters.mdx (hunk 2 above)
Comment (olirice) [reply apps/www/_blog/2023-08-12-vecs-adapters.mdx:17]: > I'm wondering if they ever had to touch a vector previously?

yes, without adapters the user was responsible for encoding the vector and then passing the vector to `vecs`

**Comment by olirice** [comment]
closing in favor of HF article

**Pull request closed without merging**